tests/sceneManager_test.cpp

```diff
@@ -3,14 +3,64 @@
 #include "obj/objects.h"
 #include "gtest/gtest.h" // https://google.github.io/styleguide/cppguide.html#Namespaces
 
+// TODO: change underscore to the end
+
 using core::Ray;
 using core::RayHitData;
 using core::Vec3;
 using generators::PointSource;
 using objects::Object;
 using objects::TriangleObj;
 
-const double kSkipFreq = 1000;
 generators::FakeRandomGen fakeRand;
 
-// TODO: Change all doubles to float
+class SceneManagerTest : public ::testing::Test
+{
+protected:
+    SceneManager manager;
+    double energyCollectorRadius = objects::EnergyCollector(Vec3()).getRadius(); // TODO: I dont like this. Vec3 should be skip ;-)
+                                                                                 // TODO: delete hard dependency
+    bool performHitAtEnergyCollectors(const Ray &tempRay, RayHitData *hitData)
+    {
+        for (objects::EnergyCollector *energyCol : manager.getEnergyCollectors())
+        {
+            const double kSkipFreq = 1000;
+            if (energyCol->hitObject(tempRay, kSkipFreq, hitData))
+                return true;
+        }
+        return false;
+    }
+};
+
+TEST_F(SceneManagerTest, PrintAllEnergyCollectorpositions) // TODO: Remove when second test is passed
```

Comment:
Why can't you just move this code to the beginning of the second test?

Reply:
done